app/assets/stylesheets/_variables.scss

```diff
@@ -26,29 +21,14 @@ $dark-gray: #333;
 $medium-gray: #999;
 $light-gray: #DDD;
 
-// Background Color
-$base-background-color: white;
-
 // Font Colors
 $base-font-color: $dark-gray;
-$base-accent-color: $blue;
-
-// Link Colors
-$base-link-color: $base-accent-color;
-$hover-link-color: darken($base-accent-color, 15);
-$base-button-color: $base-link-color;
-$hover-button-color: $hover-link-color;
+$action-color: $blue;
```

Comment:
I think changing the colors should go in to another PR. @Magnus-G if you feel strongly about this naming please do it and we can have a discussion there.